policyengine_us/parameters/gov/states/vt/tax/income/agi/retirement_income_exemption/threshold/income.yaml

```diff
@@ -0,0 +1,30 @@
+description: Vermont qualifies filers for the retirement income exemption with income at or above this threshold, based on filing status.
```

Comment:
Isn't it the reverse? This is the income limit, and they have to be below it to be eligible?

Reply:
Hi max, you are right. The variable logic is right but the description here is a mistake. Sorry for that. Will change it in the next commit.